packages/react/src/components/FilterTags/FilterTags.test.jsx

```diff
@@ -62,8 +68,20 @@ describe('Filtertags', () => {
     );
     expect(screen.getByTestId('filter-tag-container')).toBeDefined();
     expect(screen.getByTestId('filter_tags-overflow-menu')).toBeDefined();
+    jest.spyOn(console, 'error').mockImplementation(() => {});
     userEvent.click(screen.getByTestId('filter_tags-overflow-menu'));
-    expect(screen.getByTestId('filter_tags-overflow-menu-item-0')).toBeDefined();
+    await waitFor(() => {
+      expect(screen.getByTestId('filter_tags-overflow-menu-item-0')).toBeVisible();
+    });
+    expect(console.error).toHaveBeenCalledWith(
+      expect.stringContaining(
+        'Warning: validateDOMNesting(...): %s cannot appear as a descendant of <%s>.%s'
+      ),
+      '<button>',
+      'button',
+      expect.stringContaining('FilterTags')
```

Comment:
this error didn't occur until the overflow menu is opened, so I needed to add the mocked getBoundingClientRect above to make that happen.